src/styles/main.scss

```diff
@@ -1,3 +1,20 @@
-body {
-  margin: 0;
+@import "./blocks/header.scss";
+@import "./blocks/cards.scss";
+@import "./utils/reset.scss";
+@import "./utils/variables.scss";
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  max-width: $row-length;
+  padding-top: 50px;
+}
+
+.cards > :not(:nth-child(4n)) {
+  margin-right: 50px;
+}
+
+.cards > :nth-child(-n+4) {
+  margin-bottom: 48px;
```

Comment:
Check the 9th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)